src/config/AppConfiguration.vanilla.js

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+// To configure your app see:
+// https://github.com/Cosmo-Tech/azure-sample-webapp/blob/main/doc/appConfiguration.md
+
+// App version
+export const APP_VERSION = process.env.REACT_APP_VERSION;
+
+// App translation configuration
+export const LANGUAGES = {};
+
+export const FALLBACK_LANGUAGE = ''; // Language used if detection fails
+
+// Duration of inactivity after which users are automatically logged out (minutes)
+export const SESSION_INACTIVITY_TIMEOUT = 30;
+
+// Polling delay to update running scenario status (milliseconds)
+export const SCENARIO_STATUS_POLLING_DELAY = 10000;
+
+// Polling delay to update Power BI information (milliseconds)
+export const POWER_BI_INFO_POLLING_DELAY = 10000;
+
+// Type of logs to download for a scenario run (SIMPLE_LOGS or CUMULATED_LOGS)
+export const SCENARIO_RUN_LOG_TYPE = '';
```

Comment:
In order to make the first configuration easier for solutions, we can use the same value as the one we're using in the sample webapp for this parameter: new solutions can use "CUMULATED_LOGS" by default.
`export const SCENARIO_RUN_LOG_TYPE = 'CUMULATED_LOGS';`